docs/appium-capabilities.md

```diff
@@ -1,7 +1,7 @@
 ---
-id: real-devices
-title: Appium for Real Devices CLI Reference
-sidebar_label: Real Devices
+id: appium-capabilities
+title: Additional Appium Capabilities
+sidebar_label: Appium Capabilities
```

Comment:
Hi @titusfortner , could you please take a look at [this page](https://docs.staging.saucelabs.net/appium-capabilities/index.html)? It contains Appium capabilities that are referenced in the Wiki Docs, but are not included in Test Configuration Options.

I've pulled this out of the Docs tree for now while we resolve the discrepancies.